src/app/parcels/ParcelsPage.tsx

```diff
@@ -338,6 +346,57 @@ const buildLastStatusFilter = async (): Promise<Filter<ParcelsTableRow, string[]
     });
 };
 
+const buildPackingSlotFilter = async (): Promise<Filter<ParcelsTableRow, string[]>> => {
+    const packingSlotSearch = (
+        query: PostgrestFilterBuilder<Database["public"], any, any>,
+        state: string[]
+    ): PostgrestFilterBuilder<Database["public"], any, any> => {
+        const newState = state.map((state) => {
+            return state.replace(" (inactive)", "");
```

Comment:
I'm not super comfortable with this string replace, because this logic is somewhat separated from the logic that adds the " (inactive)" to the item name.

I think we should instead have key-value pairs for the items. i.e. I think `optionsSet` should be `{ key: string, value: string}[]` where key could be "Slot 1" and value could be "Slot 1 (inactive)". This way we
- only need to define the logic of adding " (inactive)" but not of removing it
- can always use the key to interact with the db

lmk if this is unclear and we can chat